posydon/unit_tests/utils/test_common_functions.py

```diff
@@ -556,6 +560,21 @@ def test_roche_lobe_radius(self, capsys):
             assert totest.roche_lobe_radius(m, 1.0)\
                    + totest.roche_lobe_radius(1.0, m) < 1.0
 
+    def test_check_for_RLO(self):
+        # missing argument
+        with raises(TypeError, match="missing 6 required positional "\
+                                     +"arguments: 'm1', 'r1', 'm2', 'r2', "\
+                                     +"'separation', and 'tolerance'"):
+            totest.check_for_RLO()
+
+        # examples
+        tests = [(1.0, 1.0, 1.0, 1.0, 1.0, 1e-3, True),
+                 (1.0, 1.0, 1.0, 1.0, 4.0, 1e-3, False)]
```

Comment:
Add tests, where only one star fills RL:
```suggestion
        tests = [(1.0, 1.0, 1.0, 1.0, 1.0, 1e-3, True),
                 (2.0, 1.0, 1.0, 1.0, 2.5, 1e-3, True),
                 (1.0, 1.0, 2.0, 1.0, 2.5, 1e-3, True),
                 (1.0, 1.0, 1.0, 1.0, 4.0, 1e-3, False)]
```